storybookjs/storybook
# Add chromatic visual tests

Issue: We didn't have visual regression tests for our stories

Note: this is built on https://github.com/storybooks/storybook/pull/2504

## What I did

Added [chromatic](https://blog.hichroma.com/introducing-chromatic-ui-testing-for-react-c5cc01a79aaa) to our CI

## How to test

Make a UI change, check out the build at https://chromaticqa.com

<strike>**NOTE**: this is not currently accessible to members of the `storybooks` org, just sorting out org permissions.</strike>

Anyone in the `storybooks` org on GH should be able to access the project here: https://www.chromaticqa.com/builds?appId=5a375b97f4b14f0020b0cda3
